space-cli/cmd/utils/transport.go

```diff
@@ -0,0 +1,81 @@
+package utils
```

Comment:
This file should be in a package called **transport**
which  will be  a folder under **utils**